Sources/Purchasing/StoreKit2/StoreKit2TransactionListener.swift

```diff
@@ -48,20 +54,20 @@ class StoreKit2TransactionListener {
         self.taskHandle = nil
     }
 
-    /// - Returns: whether the user cancelled
+    /// - Returns: `nil` `CustomerInfo` if purchases were not synced
```

Comment:
this would only happen right now on a `userCancelled`, right?

Reply:
Or if the `delegate` is `nil`.

Reply:
right. Just to make sure I'm understanding the logic correctly, this is a delegate under our control, right? meaning that in practice it really shouldn't be `nil`, and only `userCancelled` would render a `nil`?

Reply:
Yeah that's right.